actions buttons do not work in stacks view

<!--

Thanks for reporting a bug for Portainer !

Do you need help or have a question? Come chat with us on Slack http://portainer.io/slack/ or gitter https://gitter.im/portainer/Lobby.

Before opening a new issue, make sure that we do not have any duplicates
already open. You can ensure this by searching the issue list for this
repository. If there is a duplicate, please close your issue and add a comment
to the existing issue instead.

Also, be sure to check our FAQ and documentation first: https://portainer.readthedocs.io
-->

**Bug description**

In stacks view, clicking on an action in the actions column leads to a js error
```
app.c7ffb2f9.js:16 Error: Param values not valid for state 'docker.containers.container.logs'. Invalid params: [ id ]
```
**Expected behavior**

should open logs or console when clicked

**Steps to reproduce the issue:**

Steps to reproduce the behavior:
1. Go to stacks
2. Open the the container list
3. click on an action

**Technical details:**

* Portainer version: 1.19.1 (and older too)
* Docker version (managed by Portainer): 18.03
* Platform (windows/linux):linux
* Command used to start Portainer: swarm
* Browser: chrome osx

--- a/app/portainer/views/stacks/edit/stackController.js
+++ b/app/portainer/views/stacks/edit/stackController.js
@@ -233,9 +233,9 @@ function ($q, $scope, $state, $transition$, StackService, NodeService, ServiceSe
   function loadExternalSwarmStack(name) {
     var agentProxy = $scope.applicationState.endpoint.mode.agentProxy;
 
-    retrieveSwarmStackResources(name)
+    retrieveSwarmStackResources(name, agentProxy)
     .then(function success(data) {
-      assignSwarmStackResources(data);
+      assignSwarmStackResources(data, agentProxy);
     })
     .catch(function error(err) {
       Notifications.error('Failure', err, 'Unable to retrieve stack details');

